pkg/networkservice/common/updatepath/client.go

```diff
@@ -20,44 +20,35 @@ import (
 	"context"
 
 	"github.com/golang/protobuf/ptypes/empty"
-	"github.com/networkservicemesh/api/pkg/api/networkservice"
 	"google.golang.org/grpc"
 
+	"github.com/networkservicemesh/api/pkg/api/networkservice"
+
 	"github.com/networkservicemesh/sdk/pkg/networkservice/core/next"
-	"github.com/networkservicemesh/sdk/pkg/tools/token"
 )
 
-type updatePathClient struct {
-	commonUpdatePath
+type idClient struct {
+	name string
 }
 
-// NewClient - creates a NetworkServiceClient chain element to update the Connection.Path
-//             - name - the name of the NetworkServiceClient of which the chain element is part
-func NewClient(name string, tokenGenerator token.GeneratorFunc) networkservice.NetworkServiceClient {
-	return &updatePathClient{
-		commonUpdatePath: commonUpdatePath{
-			name:           name,
-			tokenGenerator: tokenGenerator,
-		},
-	}
+// NewClient - creates a new setId client.
```

Comment:
Comment needs to also shift to reflect the new package name

Reply:
Fixed.